force-app/tests/Shared Code/Log_Tests.cls

```diff
@@ -0,0 +1,105 @@
+@isTest
+private class Log_Tests {
+    @isTest
+    static void testDirectPublishMethodsPositive() {
+        Test.startTest();
+        Log.get().publish('This is a test');
+        Log.get().publish(new DMLException('hi from exception land'));
+        Test.stopTest();
+        Test.getEventBus().deliver();
+        String currentRequestId = Request.getCurrent().getRequestId();
+        Quiddity currentQuiddity = Request.getCurrent().getQuiddity();
+        List<LogEvent__c> events = [
+            SELECT Id, Request_Id__c, Quiddity__c, Log_Data__c, Severity__c
+            FROM LogEvent__c
+            ORDER BY Severity__c
+        ];
+        System.assertEquals(
+            2,
+            events.size(),
+            'expected to find only a single event'
+        );
+
+        for (LogEvent__c logEvt : events) {
+            System.assertEquals(
+                currentRequestId,
+                logEvt.Request_Id__c,
+                'Expected the reuqest id to have been logged'
+            );
+            System.assertEquals(
+                currentQuiddity.name(),
+                logEvt.Quiddity__c,
+                'Expected quiddity on log message to match apex test context'
+            );
+            System.assert(
```

Comment:
Any specific reason to not be consistent here using `System.assertEquals()`? Sorry, I may not be familiar but trying to understand this.